### PR TITLE
Update styling to match theme of ASP.NET

### PR DESCRIPTION
**Pull request by shirhatti**
This also addresses https://github.com/aspnet/Docs/issues/475

### CODE REVIEW

**Comment by shirhatti** [comment]
Created a private build at https://readthedocs.com/docs/aspnet-aspnet/en/shirhatti-logo/

**Comment by danroth27** [comment]
@jongalloway could you take a look at these changes?

**Comment by danroth27** [comment]
The left nav doesn't appear to indent correctly: 
image

**Comment by danroth27** [comment]
The fonts in the left nav still seem to be picking up the font family from the Sphinx theme.

**Comment by danroth27** [comment]
The left nav selected background is showing up as #253278. I believe it should be #1e194f (a darker blue).

File: common/_static/custom.css [hunk 1]
```diff
@@ -1,27 +1,94 @@
+@font-face {
```
Comment (danroth27) [inline common/_static/custom.css]: @DamianEdwards Not sure if this approach is recommended - we don't do this on the http://asp.net site. Is this to enable consistent fonts on OS X?

**Comment by danroth27** [comment]
Breadcrumb font colors are slightly off. Should be #505050, but it's showing up as #404040.

File: common/_static/custom.css (hunk 1 above)
Comment (DamianEdwards) [inline common/_static/custom.css]: I believe it's what we do on http://get.asp.net to ensure fonts work well on OSX

Comment (DamianEdwards) [inline common/_static/custom.css]: That said, it should be hard-coded to "https://". We don't do scheme-less URLs these days.

**Comment by danroth27** [comment]
The GitHub icon isn't showing up in the Edit on GitHub link because it is being displayed as white:

image

image

**Comment by danroth27** [comment]
Top level links in the on page TOC should be 14px instead of 16px

**Comment by danroth27** [comment]
Text style isn't supposed to change on hover, but it is turning white:

image

**Comment by danroth27** [comment]
Body text should be 13px, #505050, but it's showing up as 16px, #404040. 
Footer text should also be 13px.

**Comment by danroth27** [comment]
:watch:

**Comment by shirhatti** [comment]
@danroth27 I think I captured all your feedback in this iteration.
FWIW, I think 13px for the font size is too small to comfortably read.

**Comment by shirhatti** [comment]
Oops. Looks like I missed the GitHub Logo comment. Will update

**Comment by shirhatti** [comment]
@danroth27 Can you have a look now? I think I got everything in this pass

**Comment by danroth27** [comment]
Looks correct to me.

@DamianEdwards any additional feedback?

**Comment by danroth27** [comment]
:shipit:

**Comment by shirhatti** [comment]
Merged in 2736a704b8c1dd70cd2177f98750cfe57dac6b7e

**Pull request closed without merging**